code/datums/diseases/advance/symptoms/fever.dm

```diff
@@ -51,6 +50,7 @@ Bonus
 		to_chat(M, "<span class='warning'>[pick("You feel hot.", "You feel like you're burning.")]</span>")
 	else
 		to_chat(M, "<span class='userdanger'>[pick("You feel too hot.", "You feel like your blood is boiling.")]</span>")
+	set_body_temp(A.affected_mob, A)
```

Comment:
I assume this is a bugfix of some sort?

Reply:
When i has set this up originally i put the temp change at the wrong point, it needed to be updated on activations not only at the start.